### DISCUSSION TITLE
How to give a slide up animation on AlertDialog

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#1119] [Q&A]
### Question

I have a simple question . How to animate AlertDialog and how to add bgcolor to AlertDialog thanks

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Offset animation could be used for that, I believe, but if only you "simulate" your own dialog with overlay `Container` control: https://github.com/flet-dev/examples/blob/main/python/controls/animation/animate-offset.py
